LXC App Update script fails trying to backup grist to nonexistent location while backup flag is off

### ✅ Have you read and understood the above guidelines?

yes

### 🔎 Did you run the script with verbose mode enabled?

Yes, verbose mode was enabled and the output is included below

### 📜 What is the name of the script you are using?

LXC App Update

### 📂 What was the exact command used to execute the script?

var_backup=no \ var_tags=ua \ var_container=all_running \ var_unattended=yes \ var_skip_confirm=yes \ bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/tools/pve/update-apps.sh)"

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 13

### 📈 Which Proxmox version are you on?

9.1.6

### 📝 Provide a clear and concise description of the issue.

I ran the lxc update script provided above and it failed while cleaning up (I think) the Grist update because it was trying to do something to do with backups and a directory that didn't exist (most likely because the `backup=no` flag was set).

### 🔄 Steps to reproduce the issue.

1. Have Grist 1.7.11 installed.
2. Run the script provided earlier including the environment variables from the node as root.
3. Grist will update to 1.7.12, but the script will fail after the update is complete.
4. Can't run it again in verbose mode as the update was successful.

### ❌ Paste the full error output (if available).

```
⠋ Updating Gristcp: cannot stat '/opt/grist_bak/docs/*': No such file or directory
  ✖️   in line 49: exit code 1 (General error / Operation not permitted): while executing command cp -r /opt/grist_bak/docs/* /opt/grist/docs/

  --- Last 20 lines of log ---
[2026-03-31 15:53:11] [INFO] Checking for update: grist
[2026-03-31 15:53:11] [OK] Update available: grist 1.7.11 → 1.7.12
[2026-03-31 15:53:11] [INFO] Stopping Service
[2026-03-31 15:53:12] [OK] Stopped Service
[2026-03-31 15:53:12] [INFO] Creating backup
[2026-03-31 15:53:12] [OK] Backup created
[2026-03-31 15:53:12] [INFO] Fetching GitHub release: grist (1.7.12)
[2026-03-31 15:53:16] [OK] Deployed: grist (1.7.12)
[2026-03-31 15:53:16] [INFO] Updating Grist
[2026-03-31 15:53:16] [ERROR] in line 49: exit code 1 (General error / Operation not permitted): while executing command cp -r /opt/grist_bak/docs/* /opt/grist/docs/
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/grist.sh
+++ b/ct/grist.sh
@@ -46,9 +46,11 @@ function update_script() {
     msg_info "Updating Grist"
     mkdir -p /opt/grist/docs
     cp -n /opt/grist_bak/.env /opt/grist/.env
-    cp -r /opt/grist_bak/docs/* /opt/grist/docs/
-    cp /opt/grist_bak/grist-sessions.db /opt/grist/grist-sessions.db
-    cp /opt/grist_bak/landing.db /opt/grist/landing.db
+    if ls /opt/grist_bak/docs/* &>/dev/null; then
+      cp -r /opt/grist_bak/docs/* /opt/grist/docs/
+    fi
+    [[ -f /opt/grist_bak/grist-sessions.db ]] && cp /opt/grist_bak/grist-sessions.db /opt/grist/grist-sessions.db
+    [[ -f /opt/grist_bak/landing.db ]] && cp /opt/grist_bak/landing.db /opt/grist/landing.db
     cd /opt/grist
     $STD yarn install
     $STD yarn run install:ee

